code/src/components/ToDoSummary.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { todos } from "../reducers/todos.js";
+import styled from "styled-components";
+
+export const TodoSummary = () => {
+  const dispatch = useDispatch();
+  // Get list information for this list
+  const list = useSelector((store) => store.todos.list);
```

Comment:
Since you're only using the items-array, it's better to just get that info from the store:
`const list = useSelector((store) => store.todos.list.items);`